apps/ledger-live-desktop/src/renderer/hooks/useListenToHidDevices.ts

```diff
@@ -1,19 +1,29 @@
 import { useEffect } from "react";
-import { useDispatch } from "react-redux";
+import { useDispatch, useStore } from "react-redux";
 import { Subscription, Observable } from "rxjs";
 import { DeviceModelId } from "@ledgerhq/types-devices";
+import { useDeviceManagementKit, DeviceManagementKitTransport } from "@ledgerhq/live-dmk";
+import { getFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { IPCTransport } from "~/renderer/IPCTransport";
 import { addDevice, removeDevice, resetDevices } from "~/renderer/actions/devices";
-import { IPCTransport } from "../IPCTransport";
+import { overriddenFeatureFlagsSelector } from "~/renderer/reducers/settings";
+import { State } from "~/renderer/reducers";
 
 export const useListenToHidDevices = () => {
   const dispatch = useDispatch();
+  const store = useStore<State>();
+  const localOverrides = overriddenFeatureFlagsSelector(store.getState());
+  const ldmkFeatureFlag = getFeature({ key: "ldmkTransport", localOverrides });
```

Comment:
[SHOULD] Here you can just do `useFeature("ldmkTransport")` since you're in a react context, with the FF provider & the store at the root of the app, the overrides will already be computed.
This logic was needed in `live-common-setup.ts` because it's not in a react context.